src/scheduler/data_types.h

```diff
@@ -194,7 +194,7 @@ struct th_data_nd_eligible
 
 struct th_data_dup_nd_info
 {
-	unsigned int err:1;
+	unsigned int error:1;
```

Comment:
Do you still want to keep it as a bit field? It will loose the flexibility of classifying errors.